centaur/src/it/scala/centaur/AbstractCentaurTestCaseSpec.scala

```diff
@@ -26,11 +28,15 @@ abstract class AbstractCentaurTestCaseSpec(cromwellBackends: List[String], cromw
   SuccessReporters.getClass
 
   private def testCases(baseFile: File): List[CentaurTestCase] = {
+    val successfullTests = TestsReportsSerializer.read(CentaurConfig.testsReports)
+    val cromwellVer = VersionUtil.getVersion("cromwell")
```

Comment:
`cromwell` should be a variable.

Reply:
Anywhere else in project `getVersion` is called with hardcoded strings as parameters (see pic. attached). I can introduce an Object with modules names, if needed. 
![pic](https://i.imgur.com/8NOw3yM.png)

Reply:
Object with values will be overkill for this case I guess.
But introducing variable instead of hardcoded string in the places we're using it is a good thing